### ISSUE TITLE
Examples in the HTML version are not fully visible without scrolling

### ISSUE THREAD

**Issue by youennf** [martinthomson/i-d-template#167]
On https://youennf.github.io/mdns-ice-candidates/draft-ietf-rtcweb-mdns-ice-candidates.html, we end up with examples being cut by default and sliding is needed.

I am wondering whether we can extend the max-size to address this.

**Comment by youennf** [comment]
Might be related to https://github.com/martinthomson/i-d-template/commit/d0b8dca9498bd91039fec15022f177327fa93dab

**Comment by martinthomson** [comment]
Those are very wide pictures, for sure, but they fix into a text document, so they are valid.  I tweaked the font size down.  My font options for monospace are limited.  Consolas is narrow enough that 15px is fine, but Courier and Courier New are too broad and I had to go down to 13.5px.  If I felt like it was possible to ship fonts, this problem goes away, but that's not realistic.  I hope that it looks better now.
